### PR TITLE
fixed Coin update error when start date is OOR

### PR DESCRIPTION
**Pull request by FrozenKiwi**
On initialization, if no coin rate exists, the DB would fail to update.

This bug only affected prodtest, but theoretically could impact release if (for example) google went down for 3+ days

### CODE REVIEW

File: __mocks__/axios/index.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import stock from './fetch.test.stock.json';
+import fx from './fetch.test.fx.json';
+
+// NOTE: This axios mock has not been used/tested outside of FinnHub/index.test.ts
```
Comment (FrozenKiwi) [inline __mocks__/axios/index.ts:4]: The mock turned out to not be useful in tracking this bug, but it is nice to have so left it in here

File: apps/rates-service/src/controllers/update.controller.ts [hunk 2]
```diff
@@ -19,4 +20,13 @@ export class UpdateController extends Controller {
           throw e;
       }
     }
+
+    /**
+     * Test exception emailing
+     */
+    @Get('throwError')
+    @Response('500', 'always throws')
+    async throwError() {
```
Comment (FrozenKiwi) [inline apps/rates-service/src/controllers/update.controller.ts:29]: will remove this function later (after testing on deployed servers)

File: apps/rates-service/src/internals/rates/fetchCoin.ts [hunk 3]
```diff
@@ -3,13 +3,19 @@ import { RateOffsetFromMarket, CoinUpdateInterval, CoinRate } from "./types";
...
   // This is to ensure we include the candle starting 1.5 mins
   // before our last validity expired
-  const fetchTimestamp = latestUntil - 6.5 * (60 * 1000);
-  return fetchNewCoinRates("1", fetchTimestamp, Date.now());
+  const now = DateTime.now();
+  // We can only fetch
+  let fetchTimestamp = Math.max(
+    latestUntil - 6.5 * (60 * 1000),
+    now.minus({days: 3}).toMillis()
+  );
```
Comment (FrozenKiwi) [inline apps/rates-service/src/internals/rates/fetchCoin.ts:17]: We seem to be limited to 3 days of history.

Comment (FrozenKiwi) [reply apps/rates-service/src/internals/rates/fetchCoin.ts:17]: (for 1-min results anyway)

**Review by MarieTheCoin — APPROVED** [review]

**Pull request merged**